apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerGridViewModel.swift

```diff
@@ -102,4 +110,20 @@ class SpeakerGridViewModel: ObservableObject {
             }
         }
     }
+    
+    private func makeSpeaker(participant: LocalParticipantManager) -> SpeakerVideoViewModel {
+        let isHost = speakersStore.host?.identity == participant.identity
+        return SpeakerVideoViewModel(participant: participant, isHost: isHost)
+    }
+
+    private func makeSpeaker(participant: RemoteParticipantManager) -> SpeakerVideoViewModel {
+        let isHost = speakersStore.host?.identity == participant.identity
+        return SpeakerVideoViewModel(participant: participant, isHost: isHost)
+    }
+}
```

Comment:
I am for sure stretching the existing architecture here to combine room and sync state. I was able to avoid adding another user abstraction for now. I suspect when I add the screen share layout a new user abstraction to help merge state from two sources may be unavoidable.

I already have `RemoteParticipantManager` and `LocalParticipantManager` for room state. `SyncUsersStore.User` for sync state. And `SpeakerVideoViewModel` for UI state. To really do this right I would probably need to create a new user abstraction with merged state that sits between `SpeakerVideoViewModel` and the other room and sync abstractions. But it is a new layer just for the single host property. So I held off for now by bending rules in a couple places.